.github/workflows/test-packages-action-windows.yml

```diff
@@ -78,9 +78,7 @@ jobs:
     runs-on:
       # We need to run on our self-hosted runners because we need proper credentials
       # for boto3 to scan through our repositories.
-      - self-hosted
-      - linux
-      - x86_64
+      - ubuntu-latest
```

Comment:
Aren't we running directly on Windows now?

Reply:
This is for the test martrix step.